File: LifeCpp/Event.cpp
#include "Event.h"

Event::Event() {};

Event::Event(Pozycja kto, Call co, Pozycja gdzie)
{
	this->kto.n = kto.n;
	this->kto.m = kto.m;
	this->co = co;
	this->gdzie.n = gdzie.n;
	this->gdzie.m = gdzie.m;
}

Call Event::getCo()
{
	return this->co;
}

Pozycja Event::getKto()
{
	return this->kto;
}

Pozycja Event::getGdzie()
{
	return this->gdzie;
}

File: LifeJava/atm/po/logika/Event.java
package atm.po.logika;

public class Event
{
    protected Pozycja kto;
    protected Pozycja gdzie;
    protected Call co;
    public Event(){};
    public Event(Pozycja kto, Call co, Pozycja gdzie)
    {
        this.kto = new Pozycja(kto);
        this.co = co;
        this.gdzie = new Pozycja(gdzie);
    }
    public Call getCo()
    {
        return this.co;
    }
    public Pozycja getKto()
    {
        return this.kto;
    }
    public Pozycja getGdzie()
    {
        return this.gdzie;
    }
}
